Src/ILGPU/IR/Values/PhiValue.cs

```diff
@@ -388,6 +389,16 @@ internal PhiValue(in ValueInitializer initializer, TypeNode type)
             // Phi values have already been mapped in the beginning
             rebuilder.Rebuild(this);
 
+        /// <summary cref="Value.Write{T}(T)"/>
+        protected internal override void Write<T>(T writer)
+        {
+            int index = 0;
+            foreach (var source in Sources)
+            {
```

Comment:
NIT: Please remove braces that are not required to match the style of the code base